### DISCUSSION TITLE
gismuの説明文の項について

### DISCUSSION THREAD

**Discussion by qq542vev** [ponjbogri/jvs_ja#2] [General]
突然の投稿失礼致します。先日、guskantさんによる2016年のgismuデータを利用して各gismuの英語版説明文と、日本語版説明文の項の数を比較しましたところ、幾つかの説明文で項の数が異なっていました。

具体的には、以下のようになります。

~~~
# 日本語版の説明には存在しない項

barna x3
bliku x3
canci x3 notesに補足説明あり
carna x4
cfine x2
cimni x3
cindu x2
clupa x2
crisa x3 脱字
ctebi x3
daplu x3
denmi x3 notesに補足説明あり
dirce x3
dirgo x3 notesに補足説明あり
djine x4
fonmo x3
forca x4
frati x4
gluta x2
jipno x3
karbi x5 notesに補足説明あり
klupe x3
klupe x4
kojna x3
konju x3
lanbi x3
latna x3 notesに補足説明あり
latna x4 notesに補足説明あり
lenjo x4 marimo氏の説明では項あり
macnu x3
mlana x3
mlana x4
mruli x4
natfe x3
nilce x3
pastu x2 脱字
pinsi x3
plixa x4
ranmi x4
rutni x2
sicpi x2 experimental-gismu
sigja x3
silka x2
solri x3
stagi x3
staku x4
tagji x4
tinbe x3
vidni x2
vraga x4
xanka x3 notesに補足説明あり

# 英語版の説明には存在しない項

cakla x2
frumu x2 notesの補足説明を定義に格上げ
tunlo x2 notesの補足説明を定義に格上げ
xutla x2
~~~

実際にjbovlasteにて各説明文を目視で確認しましたところ、誤字によるものもありましたが、意図的に中間の項や末尾の項の省いたと思われるものもありました。

notesに補足説明があるものや、frumu, tunloを修整するかについては、議論があるかと思いますますが、それ以外の説明文については、できれば修整されることを望んでおります。

当方は英語がとても不得意で、それに加えてロジバン初心者ということもあり、全ての説明文を修整するのは、残念ながら当方の手に余ります。

誠に恐れ入りますが、どなた様か説明文を修整していただけないでしょうか。宜しくお願い致します。

**Comment by guskant** [comment]
調査ありがとうございます。修正作業を始めます。

**Comment by qq542vev** [comment]
早速ご対応いただきましてありがとうございます。cmavoについても同様の調査を行って見たのですが、少ないながらも発見できました。

~~~
# 日本語版の説明に存在しない項

cu'o x1 項の説明はあるが、x1という形式ではない
cu'o x2 項の説明はあるが、x2という形式ではない
mei x2
mei x3
mei x4
si'e x1
si'e x2
~~~

もし、宜しければこちらの説明文についても修整していただければ、誠に幸いでございます。

**Reply by niftg** [reply]
mei4がよくわからないので質問して大全を参照してみたところ、大全の第18章の「特別なメクソ・セルブリ」節にサラッと書いてある件が関係あるかもしれません。

> この追加された場所（「-mei」の場合は x4 で、 fo が付く）は、「ma'i」のような BAI タグの代わりに使われる。なぜなら、判断基準の特定は「充分」のような主観的な単語の意味に本質的だからである。
> この場所は、主観的な数の1つを数として直接扱うときは、必ずしも明示されない。従って、「du'e ratcu」は「多すぎる鼠」を意味し、判断基準は特定されていない。

そんなわけで、当初の辞書には逐一記載することを避けたのかもしれませんが、正直よくわかりません。

**Reply by niftg** [reply]
つまり`officialdata`の中身に変更が見られる場合で、必ずしも取り込まなくてもよい変更もある気がするんですよね

**Reply by qq542vev** [reply]
ご返信頂きましてありがとうございます。個人的な見解ですが、当方もmeiのx4は取り込まなくてよいように思えます。

他のMOI類ではmeiのx4に相当するものは定義されていないようですし。

**Reply by niftg** [reply]
誰かが気まぐれにofficialdataを改竄して追加したんですかねえju'ocu'i.iacu'i（試しに自分のアカウントで開いてみたら、officialdataの編集画面も普通に出せたので）

**Reply by qq542vev** [reply]
返信ありがとうございます。当方jbovlasteの仕様については詳しくないのですが、誰でもofficialdata名を騙って編集することが可能なのでしょうか。

**Reply by guskant** [reply]
ご指摘の gismu / cmavo の日本語定義が officialdata の翻訳になるように修正し、問題点があるものについては notes 欄に補足しました。以下のリストにも反映されています。
https://guskant.github.io/lojbo/gismu-cmavo.html
更に問題があればご指摘ください。

**Reply by niftg** [reply]
ki'e sai ba'o mutce gunka .io

もし仮に個別の問題などあれば新たにスレを立ててみます.airu'e

**Reply by qq542vev** [reply]
この度は当方の突然の投稿にも関わらず、素早い更新作業ありがとうございます。😄

**Comment by guskant** [comment]
jbovlaste は誰でもどの単語も編集できます。official data について他の単語と違う点は、 vote information がデフォルトで 100000 であることです。これが 100000 のままであれば、誰も編集していません。誰かが編集すると、1人編集するごとに vote information も 1 増えます。 あるいは編集しなくても誰かが upvote マークを押すと、やはり1人あたり 1 増えます。 mei の vote が 100003 なのは、「誰かが編集したか、あるいは upvote を押した」ということが3人分あった結果のはずです。

**Reply by qq542vev** [reply]
ご返信ありがとうございます。

当方の頭の中で纏めると、「created by (自分以外のアカウント名)」と表示されている単語の説明でも、アカウントがあればその単語の説明を自由に編集できるということですよね。そして、vote informationが+1されるのですね。

これはこの議論の本題とはかけ離れてしまうのですが、編集された説明文と編集される前の説明文の差分を表示する機能はjbovlasteにはないのでしょうか。この機能がないと、何処をどの様に編集されたか、さっぱり分からないです。

**Reply by niftg** [reply]
> 編集された説明文と編集される前の説明文の差分を表示する機能はjbovlasteにはないのでしょうか。この機能がないと、何処をどの様に編集されたか、さっぱり分からないです。

現状、公開の機能には無いはずです（変更はさておき新規追加内容を逐一その追加者にメール送付する機能はあった気はする）
